module2/src/com/epam/sorting/SortingImplementer.java

```diff
@@ -0,0 +1,76 @@
+package com.epam.sorting;
+
+public class SortingImplementer {
+
+    //task 1
+    public static int[] InsertArrayIntoArray(int[] longArray, int[] shortArray, int insertIndexIntoArray) {
+        int [] summaryArray = new int[longArray.length + shortArray.length];
+
+        for (int i = 0; i < (summaryArray.length - shortArray.length + 1); i ++) {
+            if ( i < insertIndexIntoArray) {
+                summaryArray[i] = longArray[i];
+            } else if (i == insertIndexIntoArray) {
+                for (int j = 0; j < shortArray.length; j++) {
+                    summaryArray[i + j] = shortArray[j];
+                }
+            } else if (i > insertIndexIntoArray) {
+                summaryArray[i + shortArray.length - 1] = longArray[i - 1];
+            }
+        } return  summaryArray;
+    }
+
+    // task 2
+    public static int[] applySelectionSort(int[] array) {
+        for (int i = 0; i < array.length-1; i++) {
+            int minimalNumber = i;
+            for (int j = i + 1; j < array.length; j++) {
+                if (array[j] < array[minimalNumber]) {
+                    minimalNumber = j;
+                }
+            }
+            int temp = array[i];
+            array[i] = array[minimalNumber];
+            array[minimalNumber] = temp;
+        } return array;
```

Comment:
Why do you have so many "} return array"? It should be
```suggestion
        } 
        return array;
```